src/stores/configStore.js

```diff
@@ -214,7 +214,7 @@ export class ConfigStore {
         value = value.split('%2C');
```

Comment:
Shouldn't we just use decodeURI here, and then it by commas? That way we won't have to do it inside the for loop